D2.5_1-5_Overview_DC_tech/D2.5_1-5_Overview_DC_tech.qmd

```diff
@@ -259,8 +272,14 @@ Example STAC:
 * [stacindex](https://stacindex.org) a collection of STAC catalogs
 
 
-### Interplanetary File System
+## Interplanetary File System
 
 The interplanetary file system <https://ipfs.io/> is a further development of the torrent system.
 It would allow to have one version of a file in a larger network and if a file is requested for  download it can be retrieved from the nearest available position. 
 IPFS uses content addressable storage to distribute the file across the network and this makes it more fault tolerant and independent of single infrastructure components. 
+
+
+# Summary
+
+As this is the first revision of this living document, the authors do admit that the list of tools presented here is far from complete. 
+The list will be extended in the future and more data formats, tools and processing services will be added. NFDI4Earth is an open community. In case you want to suggest additional tools to be added in the future, feel free to reach out to the NFDI4Earth Measure 2.5 team or start a discussion directly on [Github](https://github.com/felixcremer/NFDI4EarthM2.5/issues). 
```

Comment:
Should we add myself as a contact point for the NFDI4Earth M2.5 Team?

Reply:
Yes, this would be good